app/assets/stylesheets/components/vendor/breadcrumbs.scss

```
@@ -29,14 +29,14 @@
  */
 
 .breadcrumbs {
-  @extend %site-width-container;
+  @include govuk-width-container;
   @include core-16($line-height: (25 / 16), $line-height-640: 1.75);
 
-  padding: $gutter-one-third 0;
+  padding: govuk-spacing(4) 0;
```

`$gutter-one-third` is `10px` so this should be
```suggestion
  padding: govuk-spacing(2) 0;
```

---

Good spot, I've fixed this